watir-webdriver.gemspec

```diff
@@ -1,36 +1,37 @@
 # -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "watir-webdriver/version"
 
 Gem::Specification.new do |s|
-  s.name        = "watir-webdriver"
-  s.version     = Watir::VERSION
+  s.name        = 'watir'
+  s.version     = '6.0.0'
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Jari Bakken"]
-  s.email       = ["jari.bakken@gmail.com"]
-  s.homepage    = "http://github.com/watir/watir-webdriver"
-  s.summary     = %q{Watir on WebDriver}
-  s.description = %q{WebDriver-backed Watir}
-  s.license     = 'MIT'
+  s.authors     = ['Alex Rodionov', 'Titus Fortner']
```

Comment:
I'll definitely put it back if @jarib wants. I considered that one more "contact info," than "copyright info" (which I obviously didn't change).  His call.